src/Tests/Toolchain/ToolchainProvider.test.ts

```diff
@@ -366,4 +368,10 @@ suite("Toolchain", function () {
       });
     });
   });
+
+  teardown(function () {
```

Comment:
Thanks that remembering the teardown location issue and relocate them :-D!

For the other reviewers: https://github.com/Samsung/ONE-vscode/issues/1678